### PR TITLE
Alysohor

### PR DESCRIPTION
**Pull request by LysogorAndGmail**

### CODE REVIEW

File: themes/ixtheo2/templates/layout/layout.phtml [hunk 1]
```diff
@@ -170,12 +170,15 @@ JS;
...
+    if($routeParams['controller'] == "Search" && $routeParams['action'] == "Results") {
+      $searchResults = "searchResults";
+    }?>
     <div class="container-fluid">
         <div class="row">
             <div class="col-12 px-0 pt-5 pb-5 justify-content-center">
-                <div role="main" class="main template-dir-<?=$this->templateDir?> template-name-<?=$this->templateName?>">
-                  <?php $showContainerBlock = true; $routeParams = $this->tuefind()->getRouteParams();
+                <div role="main" class="main template-dir-<?=$this->templateDir?> template-name-<?=$this->templateName?> <?=$searchResults;?>">
+                  <?php $showContainerBlock = true;
```
Comment (mtrojan-ub) [inline themes/ixtheo2/templates/layout/layout.phtml:181]: what happened to the routeParams variable which existed in this line before the change?

File: themes/ixtheo2/templates/search/searchbox.phtml [hunk 2]
```diff
@@ -46,9 +46,9 @@
         <div class="panel-home">
             <?=$this->inlineScript(\Laminas\View\Helper\HeadScript::SCRIPT, 'TueFind2.ChangeSearchboxHeight();', 'SET')?>
 
-            <div class="visible-lg visible-md visible-sm">
+            <div class="logoBox">
```
Comment (mtrojan-ub) [inline themes/ixtheo2/templates/search/searchbox.phtml:49]: can't find any CSS classes related to logoBox and logoText. Is something missing in the PR?

File: themes/ixtheo2/templates/layout/layout.phtml (hunk 1 above)
Comment (mtrojan-ub) [reply themes/ixtheo2/templates/layout/layout.phtml:181]: ah sorry, i see, you moved it to 173

File: themes/ixtheo2/templates/search/searchbox.phtml (hunk 2 above)
Comment (mtrojan-ub) [reply themes/ixtheo2/templates/search/searchbox.phtml]: OK so now you reverted that, but the old classes  (visible-...) are missing. Is that really intended?

Comment (LysogorAndGmail) [reply themes/ixtheo2/templates/search/searchbox.phtml]: now the mobile version is working well

**Pull request merged**